packages/landing/src/components/map.tsx

```diff
@@ -117,32 +119,18 @@ export class Basemaps extends Component<unknown, { isLayerSwitcherEnabled: boole
           id: newStyleId,
           type: 'raster',
           source: newStyleId,
-          paint: { 'raster-opacity': 0 },
-        });
+          paint: {
+            'raster-opacity': 0,
+            'raster-opacity-transition': {
+              duration: layerFadeTime,
+              delay: 0,
+            },
+          },
+        } as any);
```

Comment:
would be great to remove this `any` 

can we use  `this.map.setPaintProperty(..., 'raster-opacity-transition')`